python/ray/autoscaler/_private/kuberay/autoscaling_config.py

```diff
@@ -333,6 +358,31 @@ def _get_num_gpus(
     return None
 
 
+def _get_num_tpus(
+    ray_start_params: Dict[str, str],
+    k8s_resource_limits: Dict[str, Any],
+    group_name: str,
+) -> Optional[int]:
+    """Get memory resource annotation from ray_start_params or k8s_resource_limits,
+    with priority for ray_start_params.
+    """
+
+    if "num-tpus" in ray_start_params:
```

Comment:
Are we still supporting this mode?

Reply:
I don't actually see `num-tpus` being used anywhere in `ray_start_params`, TPUs are currently just supported as a custom resource. I was thinking of adding `num-tpus` as a supported mode as part of this change, but thinking about it more that probably goes beyond the scope of this PR, so I'll just change it to check for TPUs in the custom resources instead.